### ISSUE TITLE
Raised lowercase i

### ISSUE THREAD

**Issue by greuelpirat** [JetBrains/JetBrainsMono#106]
Very strange: I set Jetbrains Mono in TortoiseHG Workbench 5.0.2. And the lowercase i is raised:

![image](https://user-images.githubusercontent.com/16866222/73255251-2516d100-41c0-11ea-8ada-60e6ff7df5fb.png)

Only in this app so far. Did not notice problems with other fonts in this app. Any ideas?

**Comment by philippnurullin** [comment]
@greuelpirat Looks like hinting bug. It's strange that this only one letter shifted. What size of the font been used?

**Comment by greuelpirat** [comment]
@philippnurullin used font size was 10pt.
I made some tests. There are no issues with 8pt or 9pt.
with 10pt there seems only i to be the problem.
with 11pt problem gets worse:

![image](https://user-images.githubusercontent.com/16866222/73360642-b01cc780-42a3-11ea-960d-467bf976944b.png)

**Comment by philippnurullin** [comment]
@greuelpirat This if 100% hinting problems. Looks like TortoiseHG Workbench 5.0.2. have some issues with it. Can you try another fonts to see if this problem in JetBrains Mono?

**Comment by greuelpirat** [comment]
I made some tests with this TortoiseHG Workbench 5.0.2. Now I found, that the issues occurs in the preview for font seletion (look at the B):

![image](https://user-images.githubusercontent.com/16866222/73435590-3dfebe00-4349-11ea-8195-cfdd97eaf99c.png)

There were no other font, where this happend.

I also set the app-font some (Consolas, Courier New, Inconsolata, Roboto Mono, Ubuntu Mono). As far as I can tell, this only happens with JetBrains Mono.
